crates/sqlbuiltins/src/functions/scalars.rs

```diff
@@ -0,0 +1,994 @@
+    "date_bin('15 minutes', TIMESTAMP '2022-01-01 15:07:00', TIMESTAMP '2020-01-01)",
+    date_bin
+}
+document! {
+    "Returns the date truncated to the specified unit.",
+    "date_trunc('day', '2020-01-01')",
+    date_trunc
+}
+document! {
+    "Returns the specified part of a date.",
+    "date_part('year', '2020-01-01')",
+    date_part
+}
+document! {
+    "Converts a string to a timestamp (Timestamp<ns, UTC>). Alias for `TIMESTAMP <string>`.",
+    "to_timestamp('2020-09-08T12:00:00+00:00')",
+    to_timestamp
+}
+document! {
+    "Converts a string to a timestamp with millisecond precision (Timestamp<ms, UTC>)",
+    "to_timestamp_millis('2020-09-08T12:00:00+00:00')",
+    to_timestamp_millis
+}
+document! {
+    "Converts a string to a timestamp with microsecond precision (Timestamp<µs, UTC>)",
+    "to_timestamp_micros('2020-09-08T12:00:00+00:00')",
+    to_timestamp_micros
+}
+document! {
+    "Converts a string to a timestamp with second precision (Timestamp<s, UTC>)",
+    "to_timestamp_seconds('2020-09-08T12:00:00+00:00')",
+    to_timestamp_seconds
+}
+document! {
+    "Converts a unix timestamp (seconds since 1970-01-01 00:00:00 UTC) to a timestamp (Timestamp<s, UTC>).",
+    "from_unixtime(1600000000)",
+    from_unixtime
+}
+document! {
+    "Compute the digest of a string using the specified algorithm. Valid algorithms are: md5, sha224, sha256, sha384, sha512, blake2s, blake2b, blake3",
+    "digest('hello', 'sha256')",
+    digest
+}
+document! {
+    "Compute the MD5 digest of a string. Alias for `digest(<string>, 'md5')`.",
+    "md5('hello')",
+    md5
+}
+document! {
+    "Compute the SHA-224 digest of a string. Alias for `digest(<string>, 'sha224')`.",
+    "sha224('hello')",
+    sha224
+}
+document! {
+    "Compute the SHA-256 digest of a string. Alias for `digest(<string>, 'sha256')`.",
+    "sha256('hello')",
+    sha256
+}
+document! {
+    "Compute the SHA-384 digest of a string. Alias for `digest(<string>, 'sha384')`.",
+    "sha384('hello')",
+    sha384
+}
+document! {
+    "Compute the SHA-512 digest of a string. Alias for `digest(<string>, 'sha512')`.",
+    "sha512('hello')",
+    sha512
+}
+document! {
+    " Encode a string using the specified encoding. Valid encodings are: hex, base64",
+    "encode('hello', 'hex')",
+    encode
+}
+document! {
+    "Decode a string using the specified encoding. Valid encodings are: hex, base64",
+    "decode('68656c6c6f', 'hex')",
+    decode
+}
+document! {
+    "Returns the Arrow type of the argument.",
+    "arrow_typeof(1)",
+    arrow_typeof
+}
+document! {
+    "Append an element to the end of an array.",
+    "array_append([1, 2], 3)",
+    array_append
+}
+document! {
+    "Concatenate two arrays.",
+    "array_concat([1, 2], [3, 4])",
+    array_concat
+}
+document! {
+    "Returns the dimensions of an array.",
+    "array_dims([[[1]]])",
+    array_dims
+}
+document! {
+    "Check if an array is empty. Returns true if empty",
```

Comment:
Most of the verbs in these string are indicative (e.g. "Returns <>" "Converts <>") but in this and the previous few the verbs are suddenly imperative (e.g. "Check <>" "Concatenate <>" "Append <>"). This specific one is odd because it is two sentences (but I think doesn't need to be we can delete the first one?)

I think indicative verbs are best for this kind of thing, but being inconsistent stands out to me and makes me think harder about if the difference means something.